services/_shared/adapters/pg_install_store.py

```
@@ -1053,6 +1054,31 @@ def list_dep_watch_repos(install_id: int) -> list[dict[str, Any]]:
 
 
+def list_reopen_watch_repos(install_id: int) -> list[dict[str, Any]]:
+    """Repo rows with reopen_watch_enabled=true (grug#730 audit finding) -
+    same store-driven targeting pattern as dep_watch/pulse."""
+    with get_pool().connection() as conn:
+        rows = conn.execute(
+            f"""
+            SELECT sk, data FROM grug_kv
+            WHERE pk = %s AND sk LIKE 'REPO#%%'
+              AND data->>'reopen_watch_enabled' = 'true' AND {TTL_LIVE}
+            """,
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)